src/Workspaces/CSharp/Portable/Classification/EmbeddedLanguages/CSharpFallbackEmbeddedLanguageClassifier.cs

```diff
@@ -2,20 +2,16 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Composition;
 using Microsoft.CodeAnalysis.Classification;
 using Microsoft.CodeAnalysis.CSharp.EmbeddedLanguages.LanguageServices;
-using Microsoft.CodeAnalysis.Host.Mef;
 
 namespace Microsoft.CodeAnalysis.CSharp.Classification
 {
-    [ExportEmbeddedLanguageClassifier(PredefinedEmbeddedLanguageClassifierNames.Fallback, LanguageNames.CSharp), Shared]
     internal class CSharpFallbackEmbeddedLanguageClassifier : AbstractFallbackEmbeddedLanguageClassifier
```

Comment:
I don't see any reason this shouldn't be given a more descriptive name. Yes its still technically a "fallback", but I don't think it would be any harder to understand if the code simply had a comment saying "if nothing else we'll use our CSharpEscapeCharacterClassifier" or something.